lib/metrics/stats.py

```diff
@@ -278,6 +287,16 @@ def ratio_max_percentile_for_period(contributions, percentage):
 
     return result
 
+def power_law_alpha_calc(data):
+    try:
+        contribs = np.array(contributions_per_author(data))
+        if (len(contribs) == 0):
+            return np.nan
+        r = powerlaw.Fit(contribs, discrete=True)
+        return r.power_law.alpha
+    except:
```

Comment:
Why does this need to be enclaused in a try...except block?

Reply:
Because the powerlaw.Fit method may throw an exception depending on the input: if the data is not enough to fit a power law candidate, for instance

Reply:
additionally, Wikichron does not handle the exceptions of the metric methods, so they should be handled there.